<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>482</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>482</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>inputFile</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: #f6e58d;</string>
  </property>
  <widget class="QPushButton" name="inputFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>461</width>
     <height>71</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 15pt &quot;이순신 돋움체 M&quot;;
border-radius: 20px;
background-color: #4b7bec;
color: rgb(255, 255, 255);
}
QPushButton:hover{
font: 15pt &quot;이순신 돋움체 M&quot;;
border-radius: 20px;
background-color: #3867d6;
color: rgb(255, 255, 255);
}
QPushButton:pressed{
background-color: rgb(255, 255, 255);
    border-style: inset;
color: rgb(0,0,0);
}
</string>
   </property>
   <property name="text">
    <string>이용할 파일 선택</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="fileName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="input_quit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;이순신 돋움체 M&quot;;
border-radius: 20px;
background-color: rgb(255, 255, 255);
color:  #4b7bec;</string>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
